tools/pfe-tools/11ty/plugins/pfe-assets.cjs

```diff
@@ -91,7 +91,7 @@ async function bundle() {
  */
 function demoPaths(content) {
```

Comment:
If we are creating 2 folders under `_sites`:  `element` and `pf-element` then the paths for demo files `demo.css` and `pf-element.js` would need to be rewritten to include the full path to the full component dir as well.

![image](https://user-images.githubusercontent.com/13666/215169201-e001837d-a686-4d2e-86db-2d675fe49743.png)

These files are currently not found as they point to the `element` folder and not the `pf-element` folder.

Reply:
it's possible that some of this code is replaced by the improved `addPassthroughCopy` in 11ty 2

Reply:
ok PTAL